src/main/java/dk/kb/netarchivesuite/solrwayback/parsers/ArcParser.java

```diff
@@ -235,7 +237,7 @@ private static void loadArcHeaderNotZipped(RandomAccessFile raf, ArcEntry arcEnt
   }
   
 
-  public static BufferedInputStream lazyLoadBinary(String arcFilePath, long arcEntryPosition) throws Exception{
+  public static BufferedInputStream lazyLoadBinary(String arcFilePath, long arcEntryPosition, long binarySize) throws Exception{
```

Comment:
Dont think it is necessary to add binarySize to the API.
Both the warc and arc parser re-parse the header to the get file-pointer to the correct position. And in this 
parsing there binarySize is calculated again.

Reply:
Nice catch. Fixed.